deploy/crds/keycloak.org_keycloaks_crd.yaml

```diff
@@ -59,6 +59,10 @@ spec:
                   description: If set to true, the Operator will create an Ingress
                     or a Route pointing to Keycloak.
                   type: boolean
+                host:
+                  description: If set, the Operator will use value of host for Ingress
+                    host instead of default value keycloak.local
```

Comment:
This description differs from the one in `keycloak_types.go`. Maybe `make code/gen` again?
I know it's already merged but I just spotted it while rebasing my own pull request.

Reply:
Created new PR to fix the mismatch. Thanks a lot for reporting the mismatch @dhx-teuto. Really appreciated.